### ISSUE TITLE
Some interfaces and properties declared in lib.dom are not being used by TS 3.9.2

### ISSUE THREAD

**Issue by Serabe** [microsoft/TypeScript#38544]
**TypeScript Version:**  3.9.2

**Search Terms:** "types not found 3.9", "TS2304", "TS4060"

**Code**

```ts
function getTracks(media: HTMLMediaElement): AudioTrackList {
    return media.audioTracks;
}
```

**Expected behavior:** AudioTrackList and property audioTracks are recognised as they were in 3.8.3

**Actual behavior:** Several errors of type TS2304 and TS4060 are being raised.

**Playground Link:**

- [Playground TS 3.8.3 (no error)](https://www.typescriptlang.org/play/?ts=3.8.3&ssl=3&ssc=2&pln=1&pc=1#code/GYVwdgxgLglg9mABAcwKZQCoCcCGEDWAzgBQC2qAJjDgFyIASGAsgDJOXUCiANquWFACUdAIIgqcbHnwsYhKIgDeAKERrEWdCCxJyVHADoc4+FIKEA3MoC+QA)
- [Playground TS 3.9.2 (error)](https://www.typescriptlang.org/play/?ssl=17&ssc=2&pln=1&pc=1#code/GYVwdgxgLglg9mABAcwKZQCoCcCGEDWAzgBQC2qAJjDgFyIASGAsgDJOXUCiANquWFACUdAIIgqcbHnwsYhKIgDeAKERrEWdCCxJyVHADoc4+FIKEA3MoC+QA)

**Related Issues:** No

**Comment by ShuiRuTian** [comment]
This change is done in https://github.com/microsoft/TSJS-lib-generator/pull/802
But it seems reasonable and might not be a bug:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43270
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43335
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43362

**Comment by Serabe** [comment]
[HTMLMediaElement.audioTracks](https://html.spec.whatwg.org/multipage/media.html#dom-media-audiotracks) Supported in Firefox >=33 (behind feature flag), Opera and Safari.
[AudioTrackList](https://html.spec.whatwg.org/multipage/media.html#audiotracklist-and-videotracklist-objects) Supported on Chrome >= 45(behind feature flag), Edge >=12, Firefox >=33 (behind feature flag), Internet Explorer >= 10, Safari.

How can it be argued that it is not standard when it is in the HTML Living spec as of today (last update was on May, 13th)?

**Comment by ShuiRuTian** [comment]
Make sense to me. So what we could do is just leave it to typescript team, for we do not exactly know the final decision?
Would it be better to remove "help wanted" label?

**Comment by orta** [comment]
Hi folks, [we document the heuristics](https://github.com/microsoft/TSJS-lib-generator#what-are-the-typescript-teams-heuristics-for-prs-to-the-dom-apis) which we use to determine if something is production ready for being added to our dom APIs. 

Being only available on just IE and Safari isn't stable enough for inclusion yet (maybe that'll change by the time 4.0 rolls around and it can be added

You can add an ambient  *.d.ts to your project with:

```
interface AudioTrackList {
     [Symbol.iterator](): IterableIterator<AudioTrack>;
 }

interface HTMLVideoElement {
  readonly audioTracks: AudioTrackList

  msFrameStep(forward: boolean): void;
  msInsertVideoEffect(activatableClassId: string, effectRequired: boolean, config?: any): void;
  msSetVideoRectangle(left: number, top: number, right: number, bottom: number): void;
  webkitEnterFullScreen(): void;
  webkitEnterFullscreen(): void;
  webkitExitFullScreen(): void;
  webkitExitFullscreen(): void;

  msHorizontalMirror: boolean;
  readonly msIsLayoutOptimalForPlayback: boolean;
  readonly msIsStereo3D: boolean;
  msStereo3DPackingMode: string;
  msStereo3DRenderMode: string;
  msZoom: boolean;
  onMSVideoFormatChanged: ((this: HTMLVideoElement, ev: Event) => any) | null;
  onMSVideoFrameStepCompleted: ((this: HTMLVideoElement, ev: Event) => any) | null;
  onMSVideoOptimalLayoutChanged: ((this: HTMLVideoElement, ev: Event) => any) | null;
  webkitDisplayingFullscreen: boolean;
  webkitSupportsFullscreen: boolean;
}

interface MediaError {
  readonly msExtendedCode: number;
  readonly MS_MEDIA_ERR_ENCRYPTED: number;
}
```

To bring them back until it's is further down the standard process 👍

**Comment by robwalch** [comment]
> You can add an ambient *.d.ts to your project with:

Tried this with typescript @4.1.4 but then I get error `TS2304: Cannot find name 'AudioTrack'.`. So what happened to the definition of `AudioTrack`?
